packages/build_packages.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# Builds packages, sets new versions, but doesn't upload anything to pypi.
+# Set DEV_RELEASE=1 to build the dev of packages.
+
+set -ex
+
+# Change to packages directory.
+cd "$(dirname "$0")"
+
+# ensure ordered by dependency dag
+PACKAGE_DIRS=(
+    util
+    objectstore
+    job_metrics
+    config
+    files
+    tool_util
+    data
+    job_execution
+    auth
+    web_stack
+    web_framework
+    app
+    webapps
+    test_base
+    test_driver
+    test_api
+)
```

Comment:
This list is duplicated from `packages/test.sh` , should we put it in a file a read it from there?